src/config.ts

```diff
@@ -108,15 +114,15 @@ export interface Config {
    * ```
    *
    * New values can be passed in to change what is returned from the authorization server
-   * depending on your specific scenario.
+   * depending on your specific scenario. Additional custom parameters can be added as well.
    *
-   * Additional custom parameters can be added as well:
+   * **Note:** You must provide the required parameters if this object is set.
    *
    * ```js
    * {
-   *   // Note: you need to provide required parameters if this object is set.
```

Comment:
This note is much clearer pulled out into the main body of the text (see screenshot in PR for the result)